whisk/activation.go

```diff
@@ -189,6 +204,44 @@ func (s *ActivationService) List(options *ActivationListOptions) ([]Activation,
 	return activations, resp, nil
 }
 
+func (s *ActivationService) Count(options *ActivationCountOptions) (*ActivationCount, *http.Response, error) {
+	// TODO :: for some reason /activations only works with "_" as namespace
```

Comment:
I know this comment is not yours (inherited from other methods) but I don't believe this is true at all. The value for the namespace generally is either `_` or a valid string representing a namespace (and typically has to match the API key otherwise it's a 401). In any case, don't need to fix it just making a historical note.